CanvasCore/CanvasCore/WhizzyWig/WhizzyWigView.swift

```diff
@@ -23,7 +23,16 @@ import Core
 public typealias URLHandler = (URL)->()
 var WhizzyWigOpenURLHandler: URLHandler? = { url in
     guard let from = UIApplication.shared.keyWindow?.rootViewController?.topMostViewController() else { return }
-    AppEnvironment.shared.router.route(to: url, from: from, options: nil)
+    let request = GetWebSessionRequest(to: url)
+    AppEnvironment.shared.api.makeRequest(request) { response, _, error in
+        DispatchQueue.main.async {
+            guard let response = response, error == nil else {
+                AppEnvironment.shared.router.route(to: url, from: from, options: nil)
+                return
+            }
+            AppEnvironment.shared.router.route(to: response.session_url, from: from, options: nil)
+        }
+    }
```

Comment:
this will change links in Student Quizzes to not use native views when they are available. Could we instead make the router fallback convert to an authenticate url if possible?

I think Student tries to do that using `ExternalToolManager.shared.showAuthenticatedURL`

Reply:
I didn't test in Student Quizzes but in Parent it still shows the native view if we have a route for it. I think it's because the url is the same except that it has a verifier token appended so the route still gets recognized. Does that sound right?

Reply:
I will test in Student Quizzes to be sure.

Reply:
@vanwagonet I moved it to the router.